packages/mdl-ripple/animation.js

```diff
@@ -2,13 +2,31 @@
  * Create a sequential animation/transition from a list of class names.
  */
 export class SequentialAnimation {
+  /**
```

Comment:
I changed my mind on this one. Let's kill all the closure annotations and references, it's not worth the effort especially since we don't have the toolchain integrated.